core/services/ocr2/plugins/threshold/decryption_queue.go

```diff
@@ -61,6 +62,10 @@ func NewDecryptionQueue(maxQueueLength int, maxCiphertextBytes int, maxCiphertex
 	return &dq
 }
 
+func toHex(ciphertextId []byte) string {
+	return "0x" + hex.EncodeToString(ciphertextId)
```

Comment:
Would it be simpler to add this as a `String()` method for `decryptionPlugin.CiphertextId`?

Reply:
Good suggestion, thanks :)

Reply:
Actually I'll address that in a separate PR.